frameworks/helloworld/tests/test_sanity.py

```diff
@@ -284,6 +284,11 @@ def fn():
 
     shakedown.wait_for(lambda: fn())
 
+    # In order to prevent the second scheduler instance from obtaining a lock, we undo the "scale-up" operation
```

Comment:
I was under the understanding that `app["labels"]` did not change, but maybe because python is pretty reference-y, the pop would have affected the original object. I will make it more explicit.